### PR TITLE
Example for MGLImageSource

### PR DESCRIPTION
**Pull request by jmkiley**
Addresses #93 and adds example for `MGLImageSource`.

- [x] Add comments.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -21,3 +20,13 @@ end
 target 'ExamplesUITests' do
   # Pods for testing
 end
+
+post_install do |installer|
+    installer.pods_project.targets.each do |target|
+        if target.name == 'DocsCode'
+            target.build_configurations.each do |config|
+                config.build_settings['SWIFT_VERSION'] = '3.0'
```
Comment (friedbunny) [inline Podfile]: The DocsCode targets should use the same Swift version as the rest of the project → #123.

File: Examples/Swift/ImageSourceExample.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        
+        // Set the corners of the image.
+        let coordinates = MGLCoordinateQuad(
+            topLeft: CLLocationCoordinate2D(latitude: 46.437, longitude: -80.425),
+            bottomLeft: CLLocationCoordinate2D(latitude: 37.936, longitude: -80.425),
+            bottomRight: CLLocationCoordinate2D(latitude: 37.936, longitude: -71.516),
+            topRight: CLLocationCoordinate2D(latitude: 46.437, longitude: -71.516))
+        
+        // Create a MGLImageSource that uses
+        let source = MGLImageSource(identifier: "radar", coordinateQuad: coordinates, url: URL(string: "https://www.mapbox.com/mapbox-gl-js/assets/radar.gif")!)
```
Comment (friedbunny) [inline Examples/Swift/ImageSourceExample.swift]: Using a remote URL is 👌, but let’s use our own copies of resources.

File: Examples/ObjectiveC/ImageSourceExample.m [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+@interface ImageSourceExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ImageSourceExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL: [MGLStyle darkStyleURL]];
```
Comment (friedbunny) [inline Examples/ObjectiveC/ImageSourceExample.m]: Nit: there’s an unnecessary space after `styleURL:`.

File: Examples/ObjectiveC/ImageSourceExample.m [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    mapView.delegate = self;
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(43.457, -75.789) zoomLevel:4 animated:NO];
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    MGLCoordinateQuad coordinates = MGLCoordinateQuadMake(CLLocationCoordinate2DMake(46.437, -80.425), CLLocationCoordinate2DMake(37.936, -80.425), CLLocationCoordinate2DMake(37.936, -71.516), CLLocationCoordinate2DMake(46.437, -71.516));
```
Comment (friedbunny) [inline Examples/ObjectiveC/ImageSourceExample.m]: Hrm, does this wrap well? Might consider doing what you did in the Swift version and manually wrap.

File: Examples/ObjectiveC/ImageSourceExample.m [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+@end
+
+@implementation ImageSourceExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle darkStyleURL]];
+    mapView.delegate = self;
```
Comment (captainbarbosa) [inline Examples/ObjectiveC/ImageSourceExample.m]: Can we add a comment above that matches our other examples?

File: Examples/ObjectiveC/ImageSourceExample.m [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Set the coordinate bounds for the raster image.
+    MGLCoordinateQuad coordinates = MGLCoordinateQuadMake(
+                                                          CLLocationCoordinate2DMake(46.437, -80.425),
+                                                          CLLocationCoordinate2DMake(37.936, -80.425),
+                                                          CLLocationCoordinate2DMake(37.936, -71.516),
+                                                          CLLocationCoordinate2DMake(46.437, -71.516));
+    
+    // Create a MGLImageSource, which can be used to add georeferenced raster images the style of a map.
```
Comment (captainbarbosa) [inline Examples/ObjectiveC/ImageSourceExample.m]: `add georeferenced raster images the style of a map.`
👇 
`add georeferenced raster images to a map.`

File: Examples/Swift/ImageSourceExample.swift [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+@objc(ImageSourceExample_Swift)
+
+class ImageSourceExample: UIViewController, MGLMapViewDelegate {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let mapView = MGLMapView(frame: view.bounds, styleURL: MGLStyle.darkStyleURL())
+        mapView.delegate = self
```
Comment (captainbarbosa) [inline Examples/Swift/ImageSourceExample.swift]: Same as in the Objective-C example.

File: Examples/Swift/ImageSourceExample.swift [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Set the coordinate bounds for the raster image.
+        let coordinates = MGLCoordinateQuad(
+            topLeft: CLLocationCoordinate2D(latitude: 46.437, longitude: -80.425),
+            bottomLeft: CLLocationCoordinate2D(latitude: 37.936, longitude: -80.425),
+            bottomRight: CLLocationCoordinate2D(latitude: 37.936, longitude: -71.516),
+            topRight: CLLocationCoordinate2D(latitude: 46.437, longitude: -71.516))
+        
+        // Create a MGLImageSource, which can be used to add georeferenced raster images the style of a map.
```
Comment (captainbarbosa) [inline Examples/Swift/ImageSourceExample.swift]: `add georeferenced raster images the style of a map.`
👇 
`add georeferenced raster images to a map.`

**Pull request merged**